docs/src/examples/components/Icon/Variations/IconExampleColor.shorthand.tsx

```diff
@@ -3,59 +3,66 @@ import * as _ from 'lodash'
 import { Flex, Icon, Grid, Text, ProviderConsumer } from '@stardust-ui/react'
 
 const IconExampleColor = () => (
-  <Grid columns="repeat(4, auto)" styles={{ alignItems: 'center' }} variables={{ gridGap: '10px' }}>
+  <Grid
+    columns="repeat(4, auto)"
+    styles={{
+      alignItems: 'center',
+    }}
+    variables={{
+      gridGap: '10px',
+    }}
+  >
     <Text content="INHERITED COLOR:" weight="bold" />
-    <Flex gap="gap.smaller" style={{ color: 'violet' }}>
+    <Flex
+      gap="gap.smaller"
+      styles={({ theme: { siteVariables } }) => ({
+        color: siteVariables.colorScheme.brand.foreground,
+      })}
+    >
       <Icon name="calendar" bordered />
       <Icon name="call" bordered />
       <Icon name="call-video" bordered />
     </Flex>
     <Text content="INHERITED COLOR FOR OUTLINED ICONS:" weight="bold" />
-    <Flex gap="gap.smaller" style={{ color: 'yellowgreen' }}>
+    <Flex
+      gap="gap.smaller"
+      styles={({ theme: { siteVariables } }) => ({
+        color: siteVariables.colorScheme.brand.foreground,
+      })}
+    >
       <Icon name="calendar" bordered outline />
       <Icon name="call" bordered outline />
       <Icon name="call-video" bordered outline />
     </Flex>
     <Text weight="bold">
-      USING THE <code>color</code> VARIABLE:
+      USING THE <code>color</code> PROP:
     </Text>
     <Flex gap="gap.smaller">
-      <Icon name="calendar" bordered variables={{ color: 'violet' }} />
-      <Icon name="call" bordered variables={{ color: 'yellowgreen' }} />
-      <Icon name="call-video" bordered variables={{ color: 'cornflowerblue' }} />
+      <ProviderConsumer
+        render={({ siteVariables: { emphasisColors, naturalColors } }) =>
+          _.keys({ ...emphasisColors, ...naturalColors }).map(color => (
+            <React.Fragment key={color}>
+              <Icon color={color} name="calendar" title={color} />
+            </React.Fragment>
```

Comment:
We don't need `Fragment` there:
`<Icon color={color} key={color} name="calendar" title={color} />`

Reply:
Daaah, thanks :)